### DISCUSSION TITLE
Manual macro placement

### DISCUSSION THREAD

**Discussion by msingh9** [The-OpenROAD-Project/OpenROAD#3080] [Q&A]
I want to do manual placement of all my srams in the design. Can someone guide me through the steps? I can use "macro_placement" command to do the placement but results are not good. I want to try manual placement. Is there a command to move the macro from one place to another by looking at GUI?

**Comment by vijayank88** [comment]
@msingh9 
Are you using ORFS?

**Reply by vijayank88** [reply]
Install OpenROAD-flow-scripts and refer the microwatt [example](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/sky130hd/microwatt/microwatt.macro_placement.cfg) and respective `config.mk` to place macros at specific location.

**Reply by msingh9** [reply]
This has been very useful. Thank you for the pointer.
